src/Renderer/ThreeExtented/FeatureToolBox.js

```diff
@@ -0,0 +1,552 @@
+/**
+ * Generated On: 2016-09-28
+ * Class: FeatureToolBox
+ * Description:
+ */
+
+import * as THREE from 'three';
+import CVML from 'Core/Math/CVML';
```

Comment:
Yes definitely. A nice Earcut package import could do the job. (Three js ExtrudeGeometry would need a bit of rewriting for our need as it does too much creating top and bottom faces)